crates/nu-command/src/conversions/into/int.rs

```diff
@@ -41,7 +41,29 @@ impl Command for SubCommand {
                 (Type::Filesize, Type::Int),
                 (Type::Table(vec![]), Type::Table(vec![])),
                 (
-                    Type::List(Box::new(Type::Any)),
+                    Type::List(Box::new(Type::String)),
+                    Type::List(Box::new(Type::Int)),
+                ),
+                (
+                    Type::List(Box::new(Type::Number)),
+                    Type::List(Box::new(Type::Int)),
+                ),
+                (
+                    Type::List(Box::new(Type::Bool)),
+                    Type::List(Box::new(Type::Int)),
+                ),
+                // Unix timestamp in nanoseconds
+                (
+                    Type::List(Box::new(Type::Date)),
+                    Type::List(Box::new(Type::Int)),
+                ),
+                (
+                    Type::List(Box::new(Type::Duration)),
+                    Type::List(Box::new(Type::Int)),
+                ),
+                // TODO: Users should do this by dividing a Filesize by a Filesize explicitly
```

Comment:
not sure i understand this :confused: 

what does the user have to do concretely?
there is a TODO, is there something to change in the code?

Reply:
(Copied the comment from the scalar case above.)

I think this raises the concern that the `into int` type erasure for `Filesize` may have unclear semantics and disallowing it would force people to use a more explicit solution. (but I am not sure I agree with that judgement as specifying that it always returns bytes would be less surprising than the rules for `Duration`)

Reply:
Removed the comment, if people care about it they should make an issue or PR.